src/clj/clojuredocs/pages/quickref/static.clj

```diff
@@ -230,7 +275,7 @@
          :title "Type Tests"})}
       {:title "Vectors",
        :groups
-       ({:syms (vec vector vector-of), :title "Create"}
+       ({:syms (vec vector vector-of mapv filterv), :title "Create"}
```

Comment:
I'm thinking these should go under 'Sequences > Use' to match up with `map` and `filter`. I know these work specifically on vectors, but my hunch is that people will be looking for the use case first and type second.